src/locale/languages/en.js

```diff
@@ -4,6 +4,9 @@
 export const en = {
   auth: {
     login: {
+      connectingToGitHub: 'Connecting to GitHub..',
+      preparingGitPoint: 'Preparing GitPoint..',
```

Comment:
Add an extra period to these: `Connecting to GitHub...` and `Preparing GitPoint...`.

Reply:
@machour and more you need to add new keys to other languages, otherwise there will be errors.

Reply:
@andrewda thanks, fixed!
@lex111 currently if a string is missing, it's displayed with an error `Missing translation for tr.auth.login.cancel`. Isn't that good enough while working on master? Or should I introduce the keys in all language with the english sentence?

Reply:
@machour yes, you need to add English sentences to all languages, just so that there are no such errors.

Reply:
@lex111 done buddy! I think I'll tackle down i18n right after auth & markdown 😄